delete_non_clonable.ts

```diff
@@ -88,11 +89,15 @@ export default function deleteNonClonable(obj: any, depth: number = 0) {
     case 'XMLHttpRequest':
       const newXHR: XHRObject = {
         readyState: obj.readyState,
-        responseText: obj.responseText,
         status: obj.status,
         statusText: obj.statusText
       };
 
+      if (obj.responseType === 'arraybuffer' || obj.responseType === 'blob')
+        newXHR.response = obj.response;
```

Comment:
Actually, will this work for blob or arraybuffer types of objects?  Do they have a response on the object that is a string?  i.e. is obj.response defined?  Where is this documented?